docker/docker-python/README.md

```diff
@@ -39,3 +47,10 @@ Credentials can be provided in the following ways:
 * **Set Environment Variables:** Set the following environment variables which the script will pick up to login to dockerhub at runtime -
     * $MXNET_DOCKERHUB_PASSWORD
     * $MXNET_DOCKERHUB_USERNAME
+    
+
+### Using the Docker Images
+* The MXNet Python Docker images can be found here: https://hub.docker.com/r/mxnet/python/
+
+* Docker Pull Command: `docker pull mxnet/python:<image_tag>`
```

Comment:
Where can I find details about version tagging?

Reply:
What details are you looking for? The tags can be looked up in the dockerhub link I have provided above - https://hub.docker.com/r/mxnet/python/tags/
Do you want me to add any other details to the README?

Reply:
I'm only on my phone, so I might be missing some stuff. I'll leave it up to you :) sorry for the confusion

Reply:
No worries, Thanks for the review.